README.md

```diff
@@ -167,17 +167,20 @@ checkout:
 dependencies:
   post:
     # run match after Circle runs bundler and pods
-    - chruby 2.3.1 && bin/execute.sh setup --no-pods
+    - bin/execute.sh setup --no-pods
 test:
+  pre:
+    - xcrun instruments -w '547B1B63-3F66-4E5B-8001-F78F2F1CDEA7' || true
+    - sleep 15
   override:
-    - chruby 2.3.1 && bin/execute.sh test
+    - bin/execute.sh test
     - mv build/reports/* $CIRCLE_TEST_REPORTS
     - cp -r $CIRCLE_TEST_REPORTS $CIRCLE_ARTIFACTS
 deployment:
   itunes_connect:
-    branch: release
+    branch: master
     commands:
-      - chruby 2.3.1 && bin/execute.sh itunes-connect --scheme "MyProject-Prod"
+      - bin/execute.sh itunes-connect --scheme "MyProject-Prod"
```

Comment:
ok i'll put that back. i like the branch folder idea. i also want to make a couple of changes to how the schemes work with alpha/beta/prod and internal testers/external testers/app store as well as those suffix/icons on the app icon.